atomic_reactor/pulp_util.py

```diff
@@ -183,8 +183,17 @@ def create_dockpulp_and_repos(self, image_names, repo_prefix="redhat-"):
     def get_image_ids_existing(self, layers):
         return self.p.getImageIdsExist(layers)
 
-    def upload(self, filename):
-        self.p.upload(filename)
+    def upload(self, filename, repo_id):
+        # returns boolean that is was uploaded to redhat-everything
+        self.log.debug("uploading %s", filename)
+        try:
+            self.p.upload(filename, repo_id)
+            self.log.debug("to the repo %s", repo_id)
```

Comment:
I find it confusing to have the sentence split up into two log messages:

```python
try:
  self.log.debug('Uploading %s to %s repo', filename, repo_id)
  self.p.upload(filename, repo_id)
  ...
except TypeError:
  self.log.debug('Falling back to uploading %s to redhat-everything repo', filename)
  self.p.upload(filename)
  ...
```
Also, log message should come before the actual upload call.

Reply:
sure